src/qml/pages/NewChatPage.qml

```diff
@@ -9,64 +10,288 @@ import "../theme"
+            signal placeholderClicked()
+
+            anchors.fill: parent
+            model: models.discoveredContacts.proxy
+            spacing: 10
+            clip: true
+            header: contactListHeader
+            delegate: contactListComponent
+            footer: Item {
+                width: width
+                height: 20
+            }
+        }
+
+        Component {
+            id: contactListHeader
+
+            Item {
+                width: parent.width
+                height: searchResultsItem.headerHeight
+                opacity: searchResultsItem.headerOpacity
+
+                ListDelegate {
+                    id: contactListDelegate
+                    anchors.centerIn: parent
+                    width: parent.width
+                    height: defaultChatHeight
+
+                    Item {
+                        width: avatar.width
+                        height: parent.height
+
+                        Avatar {
+                            id: avatar
+                            nickname: contact
+                            anchors.verticalCenter: parent.verticalCenter
+                        }
                     }
-                    else if (event.key == Qt.Key_Escape) {
-                        reject();
+
+                    Column {
+                        Layout.fillWidth: true
+                        clip: true
+
+                        Text {
+                            color: Theme.primaryTextColor
+                            font.pointSize: UiHelper.fixFontSz(15)
+                            text:  contact
+                        }
+
+                        Text {
+                            color: Theme.secondaryTextColor
+                            font.pointSize: UiHelper.fixFontSz(12)
+                            text: "Click here to create chat"
+                            width: parent.width
+                            elide: Text.ElideRight
+                            textFormat: Text.RichText
+                        }
                     }
                 }
             }
         }
 
-        Rectangle {
-            height: 1
-            color: "gray"
-            opacity: 0.5
-            Layout.fillWidth: true
-            Layout.maximumWidth: Theme.formMaximumWidth
-            Layout.alignment: Qt.AlignHCenter
+        Component {
+            id: contactListComponent
+
+            Item {
+                width: contactListView.width
+                height: defaultChatHeight
+
+                ListDelegate {
+                    id: contactListDelegate
+                    anchors.fill: parent
+
+                    Item {
+                        width: avatar.width
+                        height: parent.height
+
+                        Avatar {
+                            id: avatar
+                            nickname: model.name
+                            anchors.verticalCenter: parent.verticalCenter
+
+                            Item {
```

Comment:
Move rounded image into Avatar component as we discussed earlier

Reply:
Made it